python/src/opendp/_lib.py

```diff
@@ -61,18 +61,19 @@ def get_lib_path(name_template, architecture):
             lib_path = get_lib_path(name_template, None)
 
         try:
-            return ctypes.cdll.LoadLibrary(lib_path)
+            return ctypes.cdll.LoadLibrary(lib_path), lib_path
         except Exception as e:
             raise Exception("Unable to load OpenDP shared library", lib_path, e)
+        
 
     elif os.environ.get('OPENDP_HEADLESS', "false") != "false":
         return None
 
     else:
         raise ValueError("Unable to find lib directory. Consider setting OPENDP_LIB_DIR to a valid directory.")
+    
```

Comment:
```suggestion
        
    elif os.environ.get('OPENDP_HEADLESS', "false") != "false":
        return None

    else:
        raise ValueError("Unable to find lib directory. Consider setting OPENDP_LIB_DIR to a valid directory.")
```
Probably don't need double linebreaks withing function... but not something that really matters, either.